doc/releases/changelog-dev.md

```diff
@@ -5,7 +5,8 @@
 <h3>New features since last release</h3>
 
 * The `RotosolveOptimizer` has been generalized to arbitrary frequency spectra
-  in the cost function.
+  in the cost function. Also note the changes in behaviour listed under _Breaking
+  changes_.
```

Comment:
```suggestion
  in the cost function. Also note the changes in behaviour listed under *Breaking
  changes*.
```'
Probably makes no difference, but I am always paranoid of the md -> rest conversion!